CHANGELOG.md

```diff
@@ -1,4 +1,44 @@
+## 1.25.0 (2022-04-11)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- [Tracking] Introduce a new fluent API `mlflow.last_active_run()` that provides the most recent fluent active run (#5584, @MarkYHZhang)
+- [Tracking] Add `experiment_names` argument to the `mlflow.search_runs()` API to support searching runs by experiment names (#5564, @r3stl355)
+- [Tracking] Add a `description` parameter to `mlflow.start_run()` (#5534, @dogeplusplus)
+- [Tracking] Add `log_every_n_step` parameter to `mlflow.pytorch.autolog()` to control metric logging frequency (#5516, @adamreeve)
+- [Tracking] Log `pyspark.ml.param.Params` values as MLflow parameters during PySpark autologging (#5481, @serena-ruan)
+- [Tracking] Add support for  `pyspark.ml.Transformer`s to PySpark autologging (#5466, @serena-ruan)
+- [Tracking] Add input example and signature autologging for Keras models (#5461, @bali0019)
+- [Models] Add support for logging models to preexisting destination directories (#5572, @akshaya-a)
+- [Models] Introduce `mlflow.diviner` flavor for large-scale [time series forecasting](https://databricks-diviner.readthedocs.io/en/latest/?badge=latest) (#5553, @BenWilson2)
+- [Models] Add `mlflow_version` field to MLModel specification (#5515, #5576, @r3stl355)
+- [Models] Add `pyfunc.get_model_dependencies()` API to retrieve reproducible environment specifications for MLflow Models with the pyfunc flavor (#5503, @WeichenXu123)
+- [Models] Add `code_paths` argument to all model flavors to support packaging custom module code with MLflow Models (#5448, @stevenchen-db)
+- [Models] Support creating custom artifacts when evaluating models with `mlflow.evaluate()` (#5405, #5476 @MarkYHZhang)
+- [Scoring / Projects] Introduce `--env-manager` configuration for specifying environment restoration tools (e.g. `conda`) Deprecate `--no-conda` (#5567, @harupy)
+- [Scoring] Support restoring model dependencies in `mlflow.pyfunc.spark_udf()` to ensure accurate predictions (#5487, #5561, @WeichenXu123)
+- [Scoring] Add support for `numpy.ndarray` type inputs to `tensorflow` predict function (#5545, @WeichenXu123)
+- [Scoring] Support deployment of MLflow Models to Sagemaker Serverless (#5610, @matthewmayo)
+- [UI] Add MLflow version to header beneath logo (#5504, @adamreeve)
+- [Artifacts] Introduce a `mlflow.artifacts.download_artifacts()` API mirroring the functionality of the `mlflow artifacts download` CLI (#5585, @dbczumar)
+- [Artifacts] Introduce environment variables for controlling GCS artifact upload/download chunk size and timeouts (#5438, #5483, @mokrueger)
+
+Bug fixes and documentation updates:
+
+- [Tracking/SQLAlchemy] Create an index on `run_uuid` for PostgreSQL to improve query performance (#5446, @harupy)
+- [Tracking] Remove client-side validation of metric, param, tag, and experiment fields (#5593, @BenWilson2)
+- [Artifacts] Add compatibility for reading `ENV` variables for GCS to prevent timeouts (#5483, @sebastian-montero)
```

Comment:
Can we remove this line since we already incorporated the associated PR into the feature changelog entry here: https://github.com/mlflow/mlflow/pull/5645/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR28 ?